src/components/implementation/no_interface/scheddev/unit_schedlib.c

```diff
@@ -49,11 +49,15 @@ printc(char *fmt, ...)
 }
 
 #define N_TESTTHDS 8
+#define WORKITERS  10000
 
 void
 test_thd_fn(void *data)
 {
+	int workiters = WORKITERS * ((int)data);
 	while (1) {
+		SPIN(workiters);
```

Comment:
I'm mainly checking the time between 2 timer interrupts. 
They are consistent and about the 100us or 1000us periods I tested in the schedlib.